### PR TITLE
Added docstrings for the file simple_functions.py

### PR DESCRIPTION
**Pull request by gurupriya05**
This PR closes issue #1

### CODE REVIEW

File: simple_functions.py [hunk 1]
```diff
@@ -1,7 +1,9 @@
 # Custom python functions
 
 def double_number(a):
+'''This function takes the input and prints the number before and after doubling it and returns the doubled number'''
```
Comment (JayashreeS11) [inline simple_functions.py]: Elaborate more regarding the parameter taken by the function. Please refer to pandas docstring.

File: simple_functions.py [hunk 2]
```diff
@@ -1,7 +1,9 @@
...
     return a+a
 
 def square_number(a):
+'''This function takes the input and prints the original number and the square of it and returns the squared number'''
```
Comment (JayashreeS11) [inline simple_functions.py]: Same as above.

**Review by JayashreeS11 — CHANGES_REQUESTED** [review]

File: simple_functions.py (hunk 2 above)
Comment (gurupriya05) [reply simple_functions.py]: Fixed the issue 0893bcdeedb6118e88ac9f278b8e9a1b9ab2a5ba

**Review by JayashreeS11 — APPROVED** [review]
You can merge it to the main branch.

**Pull request merged**